### DISCUSSION TITLE
Facing an error in running train_diffusion.py command. This is in the 2d_ldm folder, following the generative tutorial for 2D

### DISCUSSION THREAD

**Discussion by programmerwali** [Project-MONAI/MONAI#7337] [General]
When I try to run the code of 2d_ldm in generative in the tutorial repository. I followed the read me and reached to the step where I have to run this code. This is to run the train_diffusion.py file:

!python train_diffusion.py -c ./config/config_train_32g.json -e ./config/environment.json -g 1

This is the error that I get:

FileNotFoundError: [Errno 2] No such file or directory: './trained_weights/diffusion_2d/autoencoder.pt'

How to resolve the auto encoder.pt error?

I have my environment set up on Google Colab and using its GPU.

**Comment by programmerwali** [comment]
<img width="1436" alt="Screenshot 2023-12-24 at 5 59 32 PM" src="https://github.com/Project-MONAI/MONAI/assets/91944091/c9e4667c-88d7-4092-a944-99aa2be90b9d">


<img width="1170" alt="Screenshot 2023-12-24 at 6 00 15 PM" src="https://github.com/Project-MONAI/MONAI/assets/91944091/0375d834-6e43-4be2-9a06-971788fd8542">

**Comment by KumoLiu** [comment]
Hi @programmerwali, we didn't provide the pre-trained weight in the tutorial repo. You can find the weights in the MONAI Zoo repo.
https://github.com/Project-MONAI/model-zoo/blob/dev/models/brats_mri_axial_slices_generative_diffusion/large_files.yml
https://github.com/Project-MONAI/model-zoo/blob/dev/models/brats_mri_generative_diffusion/large_files.yml

Hope it helps, thanks!
